tests/config/operator_install/olsconfig.crd.azure_openai_introspection.yaml

```
@@ -34,3 +34,10 @@ spec:
       - name: bar_filter
         pattern: '\b(?:bar)\b'
         replaceWith: "openshift"
+  mcp_servers:
+    - name: openshift
+      transport: stdio
+      stdio:
+        command: python
+        args:
+          - /app-root/mcp_local/openshift.py
```

Please use relative path
```suggestion
          - mcp_local/openshift.py
```

---

Once I get the CI working, I'll test if it works. There might be some changes required given the place where we execute the code.

---

It shouldn't be different than local `make run`. But okay lets wait for CI.
meanwhile if you want you can try temporarily hardcoding in the code.

Please let @xrajesh know about this.. as this will be added in operator.